src/arrays.js

```diff
@@ -1,62 +1,70 @@
 const getNthElement = (index, array) => {
-  // your code here
+  while (index>=array.length){
+    index = index - array.length
+  }
+  return array[index];
```

Comment:
This is an interesting approach! Its not good practice to reassign function parameters like this though, as it can lead to confusing code and unwanted side effects. 
An alternative is to use the modulus operator, which I see you've used in the numbers file. Since that will return a remainder instead of a decimal when dividing, it gives a whole number (EG 5 % 3 = 2). 
Since thats what you are in effect doing here when the index is greater than the length of the array you can rewrite this as follows:
```
  return array[index % array.length];
```